### PR TITLE
fix(issues): Improve UX when highlighting Group title

### PR DESCRIPTION
**Pull request by leedongwei**
Took the fastest way out to fix this UX issue.

textselection

### CODE REVIEW

File: src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  };
+
+  render() {
+    const {title, subtitle} = getTitle(this.props.data as Event);
+
+    return subtitle ? (
+      <span style={this.props.style}>
+        <span>{title}</span>
+        <Spacer />
+        <em title={subtitle}>{'asdfasdfa/asdfasdf.asdfadsf' || subtitle}</em>
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx]:
```suggestion
        <em title={subtitle}>{subtitle}</em>
```

**Review by dashed — CHANGES_REQUESTED** [review]

File: src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+type Props = {
+  data: {
+    type:
+      | 'error'
+      | 'csp'
+      | 'hpkp'
+      | 'expectct'
+      | 'expectstaple'
+      | 'default'
+      | 'transaction';
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx]: Should this union be in app/types? It feels like something that will be used in more than one place and could be part of the event type too.

File: src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx (hunk 1 above)
Comment (leedongwei) [reply src/sentry/static/sentry/app/components/eventOrGroupTitle.tsx]: Lol omg.

**Comment by leedongwei** [comment]
Latest commit is dependent on the correct types from #17058

**Review by markstory — APPROVED** [review]

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Comment by leedongwei** [comment]
Had a long weekend. Rebased on `master` and running tests again.

**Comment by leedongwei** [comment]
1c63f7a updates acceptance test snapshops.

**Comment by leedongwei** [comment]
@markstory @dashed @doralchan @billyvg

FYI: This commit has a layout change on mobile width: the subtitle will be moved into a new-line, underneath title. Titles/subtitles that are longer than the page width will be truncated with ellipsis. Please take a look at Percy's diffs.

Not sure who owns the Event/Group page, so I want to make sure y'all are aware and get your buy-in before I merge this.

**Comment by billyvg** [comment]
I would say @matejminar and @priscilawebdev probably owns these views now? It lgtm, seems more readable this way on mobile.

**Review by priscilawebdev — APPROVED** [review]
It looks good to me 👍

**Comment by priscilawebdev** [comment]
Talking about Percy's snapshots, I would say that the changes made seem to me much better now than before 😉

**Review by matejminar — APPROVED** [review]
Looks good to me, thanks.

**Pull request merged**